fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/federation/LinkedOrganizationsFragment.kt

```diff
@@ -59,6 +59,19 @@ class LinkedOrganizationsFragment : Fragment() {
     binding.inviteOtherOrganization.setOnClickListener(invitationPage)
     binding.joinOtherOrganizationInvitation.setOnClickListener(joinButton)
 
+    // Displaying the linked organizations
+    val laos = linkedOrganizationsViewModel.getLinkedLaosMap().keys
+    displayLinkedOrganizations(laos)
+    linkedOrganizationsViewModel.doWhenLinkedLaosIsUpdated { laoId, laoMap ->
+      if (laoId == laoViewModel.laoId) {
+        CoroutineScope(Dispatchers.Main).launch {
+          val currentLaos = laoMap.keys
+          displayLinkedOrganizations(currentLaos)
+        }
+      }
+    }
+
+    linkedOrganizationsViewModel.setLinkedLaosNotifyFunction()
```

Comment:
Why calling display first with the current list and then putting a custom and complex observer that uses coroutines and not directly using a simple observer? This relates to my previous comment on how to deal with modifications using observables and subjects